### PR TITLE
Add time_ms and some usage details to the README

### PR DESCRIPTION
**Pull request by janxious**
## What / Why

Add a millisecond timer to the time command for accuracy and  way to record the value in ms.

Update the README to have an annotated set of example code that is excecutable.
## Caveats

I think it is written correctly, but I would like to support passing in a pre-computed time in addition to the function form.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -36,8 +36,61 @@ config :instrumental,
...
+
+```elixir
+defmodule ImportantThing do
+  alias Instrumental, as: I
+
+  def compute_important_thing do
+    # tell Instrumental how long it takes to run a
+    # computation so we can know if our performance
+    # is changing over time.
+    value = I.time("ImportantThing.timing", &ImportantThing.compute/0)
```
Comment (jqr) [inline README.md:53]: Should we have an example of anonymous function wrapping to show that technique or is it obvious?

**Comment by jqr** [comment]
:+1: May want to revisit the README later to make all the agents consistent but this is a nice step forward.

File: README.md (hunk 1 above)
Comment (janxious) [inline README.md:53]: That seems like a valuable thing to add.

Comment (janxious) [inline README.md:53]: Actually, I think we already had this in the examples below this. So, two steps ahead of you, but I forgot by the time we talked about it!

**Pull request merged**